slam-ws/src/nav_goal/bin/movebase_client

```diff
@@ -5,17 +5,16 @@ import rospy
 import sys
 
 if __name__ == '__main__':
-   if len(sys.argv) < 4:
-        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
-   else:
-	try:
-		x = float(sys.argv[1])
-		y = float(sys.argv[2])
-		yaw = float(sys.argv[3])
+   	if len(sys.argv) < 4:
+		   rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+   	else:
+		try:
+			x = float(sys.argv[1])
+			y = float(sys.argv[2])
+			yaw = float(sys.argv[3])
 
-        	rospy.init_node('movebase_client_py')
-        	result = nav_goal.movebase_client.movebase_client(x, y, yaw)
-        	if result:
-            		rospy.loginfo("Nav Goal successfully sent to MoveBaseAction server!")
-    	except rospy.ROSInterruptException:
-        	rospy.logerr("Error - Navigation test interrupted.")
+			rospy.init_node('movebase_client_py')
+			result = nav_goal.movebase_client.movebase_client(x, y, yaw)
+			rospy.loginfo(result)
+		except:
+			rospy.logerr("Error - Navigation test interrupted.")
```

Comment:
I `Ctrl + C` and never got this error, rather got 
```
[ERROR] [1573695862.827803, 1413.650000]: Action server not available!
[INFO] [1573695862.828434, 1413.650000]: None
```

Reply:
Yeah that happens, what would you like me to change?

Reply:
Well this error should be an `"interrupted Error"`, since I interrupted the program. "Action server not available" should log when it couldn't connect to the server

Reply:
I don't know if it can tell that it was a user that stopped the program, but I can make both error messages more generic

Reply:
I clarified error messages